input-output-hk/lace
# feat(core): show mnemonic order error in each part of the flow

# Checklist

- [ ] JIRA - \<LW-6084>

---

## Proposed solution

Solution involves removing the condition for showing the error only on the last step and adding a check to see if the user is still typing.

## Testing

1. Create a wallet flow.
2. Type the mnemonic shown in wrong order e.g. (switch word 7 with 8).
3. Should see the error.

## Screenshots


https://github.com/input-output-hk/lace/assets/27725122/33e99762-f87b-41af-a1d0-3225d9026720


---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1996/6149362213/index.html) for [4ed233dd](https://github.com/input-output-hk/lace/pull/522/commits/4ed233dda5e2fdc02c12e5a6f1716778e20327b9)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 29     | 3      | 0       | 0     | 32    | ❌     |
